### PR TITLE
Added the ability to disable default css.

### PR DESCRIPTION
**Pull request by Web-VPF**

### CODE REVIEW

File: components/Channel.php [hunk 1]
```diff
@@ -81,6 +81,12 @@ public function defineProperties()
                 'type'        => 'dropdown',
                 'group'       => 'Links',
             ],
+            'css' => [
```
Comment (LukeTowers) [inline components/Channel.php:84]: if you can rename this property to `includeStyles` instead that would be better

Comment (Web-VPF) [reply components/Channel.php]: Corrected

File: components/Channel.php [hunk 2]
```diff
@@ -81,6 +81,12 @@ public function defineProperties()
                 'type'        => 'dropdown',
                 'group'       => 'Links',
             ],
+            'includeStyles' => [
+                'title'       => 'rainlab.forum::lang.prop.include_css',
+                'description' => 'rainlab.forum::lang.prop.include_css_desc',
```
Comment (LukeTowers) [inline components/Channel.php:86]: language key should be `includeStyles` and `includeStyles_desc`

Comment (Web-VPF) [reply components/Channel.php:86]: without `prop`?

Comment (LukeTowers) [reply components/Channel.php:86]: Technically it would be better for it to be `rainlab.forum::lang.components.general.properties.includeCss`

Comment (Web-VPF) [reply components/Channel.php:86]: `rainlab.forum::lang.components.general.properties.includeCss`
or
`rainlab.forum::lang.components.general.properties.includeStyles`
?

Comment (Web-VPF) [reply components/Channel.php]: Language key corrected

File: lang/en/lang.php [hunk 3]
```diff
@@ -131,5 +131,13 @@
...
         'channel_title' => 'Channel Page',
         'channel_desc' => 'Page name to use for clicking on a channel.'
+    ],
+    'components' => [
+        'general' => [
+            'properties' => [
+                'includeStyles' => 'Enable CSS',
+                'includeStyles_desc' => 'Include css file with default styles for forum'
+            ]
+        ]
```
Comment (LukeTowers) [inline lang/en/lang.php:141]: Looking good, can you make sure you add trailing commas as well please?

Comment (Web-VPF) [reply lang/en/lang.php]: added

**Comment by LukeTowers** [comment]
LGTM! Ready for merging @bennothommo

File: components/Channel.php [hunk 4]
```diff
@@ -91,7 +97,9 @@ public function getPropertyOptions($property)
 
     public function onRun()
     {
-        $this->addCss('assets/css/forum.css');
+        if ($this->property('includeStyles')) {
```
Comment (bennothommo) [inline components/Channel.php:100]: @Web-VPF this should default to `true` if the includeStyles property is missing for some reason.
```suggestion
        if ($this->property('includeStyles', true)) {
```

File: components/Channels.php [hunk 5]
```diff
@@ -69,7 +75,9 @@ public function getPropertyOptions($property)
 
     public function onRun()
     {
-        $this->addCss('assets/css/forum.css');
+        if ($this->property('includeStyles')) {
```
Comment (bennothommo) [inline components/Channels.php:78]: @Web-VPF same here.
```suggestion
        if ($this->property('includeStyles', true)) {
```

File: components/Member.php [hunk 6]
```diff
@@ -90,7 +96,9 @@ public function getPropertyOptions($property)
 
     public function onRun()
     {
-        $this->addCss('assets/css/forum.css');
+        if ($this->property('includeStyles')) {
```
Comment (bennothommo) [inline components/Member.php:99]:
```suggestion
        if ($this->property('includeStyles', true)) {
```

File: components/Posts.php [hunk 7]
```diff
@@ -78,7 +84,9 @@ public function getPropertyOptions($property)
 
     public function onRun()
     {
-        $this->addCss('assets/css/forum.css');
+        if ($this->property('includeStyles')) {
```
Comment (bennothommo) [inline components/Posts.php:87]:
```suggestion
        if ($this->property('includeStyles', true)) {
```

File: components/Topic.php [hunk 8]
```diff
@@ -106,7 +112,9 @@ public function getPropertyOptions($property)
 
     public function onRun()
     {
-        $this->addCss('assets/css/forum.css');
+        if ($this->property('includeStyles')) {
```
Comment (bennothommo) [inline components/Topic.php:115]:
```suggestion
        if ($this->property('includeStyles', true)) {
```

File: components/Topics.php [hunk 9]
```diff
@@ -74,7 +80,9 @@ public function getPropertyOptions($property)
 
     public function onRun()
     {
-        $this->addCss('assets/css/forum.css');
+        if ($this->property('includeStyles')) {
```
Comment (bennothommo) [inline components/Topics.php:83]:
```suggestion
        if ($this->property('includeStyles', true)) {
```

**Comment by LukeTowers** [comment]
Good catch @bennothommo

**Comment by Web-VPF** [comment]
@bennothommo ok

File: lang/en/lang.php [hunk 10]
```diff
@@ -131,5 +131,13 @@
         'slug_desc' => 'The URL route parameter used for looking up the topic by its slug. A hard coded slug can also be used.',
         'channel_title' => 'Channel Page',
         'channel_desc' => 'Page name to use for clicking on a channel.'
-    ]
+    ],
+    'components' => [
+        'general' => [
+            'properties' => [
+                'includeStyles' => 'Enable CSS',
+                'includeStyles_desc' => 'Include css file with default styles for forum',
```
Comment (bennothommo) [inline lang/en/lang.php:139]:
```suggestion
                'includeStyles_desc' => 'Include the CSS files with default styles for forum',
```

File: lang/en/lang.php [hunk 11]
```diff
@@ -131,5 +131,13 @@
         'slug_desc' => 'The URL route parameter used for looking up the topic by its slug. A hard coded slug can also be used.',
         'channel_title' => 'Channel Page',
         'channel_desc' => 'Page name to use for clicking on a channel.'
-    ]
+    ],
+    'components' => [
+        'general' => [
+            'properties' => [
+                'includeStyles' => 'Enable CSS',
+                'includeStyles_desc' => 'Include the CSS files with default styles for forum',
```
Comment (bennothommo) [inline lang/en/lang.php:139]:
```suggestion
                'includeStyles_desc' => 'Include the CSS files with default styles for the forum',
```

**Comment by bennothommo** [comment]
Tested and confirmed working.

**Pull request merged**